docs/R-notebooks/CorrelationShinyApp/helpers.r

```diff
@@ -0,0 +1,323 @@
+library(covidcast)
+library(ggplot2)
+library(lubridate)
+library(plotly)
+library(dplyr)
+
+
+#' Transforms a lubridate date format into string "YYYYMMDD" ready to be fed to the Delphi API
+#' 
+#' @return list with population per county/state/msa
+#' TODO code is relying on github csv that maps state IDs to letters
+get_population <- function(){
+  population_mapping = list()
+  
+  census_url = "https://www2.census.gov/programs-surveys/popest/datasets/2010-2019/counties/totals/co-est2019-alldata.csv"
+  county_pop = read.csv(census_url) %>% 
+    transmute (geo_value = 1000*as.numeric(STATE) + as.numeric(COUNTY),
+               population = POPESTIMATE2019)
+  county_pop$geo_value <- sprintf("%05d", county_pop$geo_value)
+  population_mapping$county = county_pop
+  
+  state_pop = read.csv(census_url) %>% 
+    mutate(geo_value = as.numeric(STATE)) %>% 
+    group_by(geo_value) %>% 
+    summarise(population = sum(POPESTIMATE2019))
+  state_crosswalk <- read.csv("https://raw.githubusercontent.com/cmu-delphi/covid-19/main/geographical_scope/StateToHHS.csv?token=AHQABX4BXLJ3S3SJFSOGPFC7EBAIA") %>% 
```

Comment:
This is a private repository -- we can't use this URL in a public notebook.

The covidcast package now includes `county_census`, `state_census`, and `msa_census` data frames that have the populations you want, although we don't have HRRs in the package yet. You should probably use these for consistency.